deepdream/README.md

```diff
@@ -22,7 +22,7 @@ We will use it today to attempt to turn a picture of dinosaurs into flowers (ins
 ### 1. Create a Workspace
 Learn more at https://spell.run/docs/workspaces_overview
 
-Initialize from the git repo https://github.com/spell-examples/workspaces
+Initialize from the git repo https://github.com/spell-examples/deepdream
```

Comment:
I think this might have to be `https://github.com/spellrun/spell-examples`. That's the link to the repo, and `deepdream` is a directory inside that. Maybe we can do something like
```suggestion
Initialize from the git repo https://github.com/spellrun/spell-examples, and open the notebook file at `deepdream/dream.ipynb`
```